src/main/java/seedu/address/logic/parser/ParserUtil.java

```diff
@@ -110,6 +114,41 @@ public static Tag parseTag(String tag) throws ParseException {
         return new Tag(trimmedTag);
     }
 
+    /**
+     * Parses a {@code String name} into a {@code String name} into the correct format.
+     * @param eventName
+     * @return a String object
+     * @throws ParseException
+     */
```

Comment:
Perhaps this Javadoc could also explain the parameters, return values and exceptions